plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1377,7 +1377,7 @@ public void testDeleteWithBigintEqualityPredicate()
     public void testDeleteWithVarcharEqualityPredicate()
     {
         skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE) && hasBehavior(SUPPORTS_ROW_LEVEL_DELETE));
-        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete", "(col varchar(1))", ImmutableList.of("'a'", "'A'", "null"))) {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete_with_varchar_equality_predicate", "(col varchar(1))", ImmutableList.of("'a'", "'A'", "null"))) {
```

Comment:
that will trip you over oracle's name limits, right?

Reply:
Yes. I'll add a subsequent commit that shortens names where needed. (with a TODO).

I'm also looking at upgrading the Oracle container since the version we are using seems EOL.